# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/CanvasCaptureMediaStream.cpp
# Commit: 10d327bb5893
# Full Hash: 10d327bb5893211f030222b02456bfc11e81a943
# Author: Olli Pettay <Olli.Pettay@helsinki.fi>
# Date: 2021-05-18 21:36:28
# Regressor Bug: 1705080
# File Overlap Count: 1
# Description:
#   Bug 1705080, make MediaStreams behave as normal EventTargets, r=jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115269
# ==============================================================================

diff -r 802480d6b25b -r 10d327bb5893 dom/media/CanvasCaptureMediaStream.cpp
--- a/dom/media/CanvasCaptureMediaStream.cpp	Tue May 18 16:19:44 2021 +0000
+++ b/dom/media/CanvasCaptureMediaStream.cpp	Tue May 18 16:21:27 2021 +0000
@@ -167,7 +167,7 @@
 nsresult CanvasCaptureMediaStream::Init(const dom::Optional<double>& aFPS,
                                         nsIPrincipal* aPrincipal) {
   MediaTrackGraph* graph = MediaTrackGraph::GetInstance(
-      MediaTrackGraph::SYSTEM_THREAD_DRIVER, mWindow,
+      MediaTrackGraph::SYSTEM_THREAD_DRIVER, GetOwner(),
       MediaTrackGraph::REQUEST_DEFAULT_SAMPLE_RATE,
       MediaTrackGraph::DEFAULT_OUTPUT_DEVICE);
   SourceMediaTrack* source = graph->CreateSourceTrack(MediaSegment::VIDEO);